Genre average ratings #94

soulcramer opened the pull request:
This pull request close the same issue as #169 
close #94

soulcramer committed:
✨ Display the user average rating on a genre page for the current genre.
I've also Calculated the number of anime that the user have seen with the
current genre.

soulcramer committed:
🐛 Forgot to check if the user was not nil

akyoto commented on pages/genre/genre.go:
```diff
@@ -29,7 +38,33 @@ func Get(ctx *aero.Context) string {
+		average += item.Rating.Overall
+	}
+
+	return average / float64(len(genreItems))
```
This contains a bug where a skipped item in the above loop with `continue` would still be counted towards the length of genreItems, making the average calculation incorrect.

soulcramer committed:
🐛 Fixed the average rate to be lower than expected.

The issue was that all the anime where counted when dividing the sum of
 the ratings whereas we should have ignored them.
